build-system/pr-check/build-targets.js

```diff
@@ -188,6 +191,13 @@ const targetMatchers = {
       file.startsWith('build-system/test-configs')
     );
   },
+  [Targets.IGNORE_LIST]: (file) => {
+    return (
+      ignoreListFiles.includes(file) ||
+      file === 'build-system/tasks/check-ignore-list.js' ||
```

Comment:
It looks like this PR broke `main` due to error:
```
checks.js Running amp check-ignore-list...
[22:22:47] ERROR: Unknown task check-ignore-list
[22:22:47] ⤷ Run amp --help for a full list of tasks.
[22:22:47] ⤷ Run amp <task> --help for help with a specific task.
```

Should this line, `build-system/pr-check/checks.js`, and `build-system/tasks/check-ignore-lists.js` all update `'check-ignore-list'` to `'check-ignore-lists'`? I'm seeing the latter more presently throughout the codebase than the former, but I'm not sure if there is a purpose behind the singular and plural differentiation.

Reply:
You're right, @caroqliu. Looks like this was an incomplete rename. (Feel free to revert if it's easier.)

/cc @alanorozco